packages/persist-state/src/lib/exclude-keys.ts

```diff
@@ -0,0 +1,19 @@
+import { Store, StoreValue } from '@ngneat/elf';
+import { map } from 'rxjs/operators';
+
+export function excludeKeys<S extends Store, State extends StoreValue<S>>(
+  store: S,
+  keys: string[]
```

Comment:
make it typed please

Reply:
Tried to do my best but I am no expert in TS magic. Let me know if it's ok now.